src/storageClients/BlobStorageManager.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const Promise = require('bluebird');
+const azure = Promise.promisifyAll(require('azure-storage'));
+
+const BLOB_STORAGE_CONNECTION_STRING = process.env.BLOB_STORAGE_CONNECTION_STRING;
+const TOPIC_SEED_CONTAINER = process.env.TOPIC_SEED_CONTAINER;
+
+module.exports = {
+
+  Get: (blobName, id) => {
+    let blobSvc = azure.createBlobService(BLOB_STORAGE_CONNECTION_STRING);
+    return new Promise((resolve, reject) => {
+      blobSvc.getBlobToTextAsync(TOPIC_SEED_CONTAINER, blobName, null)
+        .then(JSON.parse)
+        .then(text => {
+          let item = text[id];
+          if (item) resolve(item);
+          else reject(null);
+        })
+        .catch(reject);
+    });
+  },
+
+  List: blobNames => {
```

Comment:
Renamed list to getTopicList. Renamed blobNames to blobPath where blobPath is something like 'siteTypes/humanitarian/topics/defaultTopics.json'